coderedcms/static/coderedcms/css/codered-admin.css

```diff
@@ -290,6 +290,14 @@ input[type='checkbox']::before, input[type='radio']::before {
     }
 }
 
+/* Display the optional banner above main content */
+
+.codered-banner {
+    position: absolute;
+}
+.codered-banner ~ .content-wrapper {
+    padding-top: calc(1.5em + 8px);
```

Comment:
These seem like oddly specific magic numbers. Is there any reasoning for them? Would it perhaps make sense to set a fixed height of the banner instead, to make it easier to work with? I would be OK with such as change as the banner is purely utility and really only designed to hold one line of text / simple HTML.

Reply:
The banner's height is the text (`1.5em`) plus the bottom and top padding (`4px` x 2) hard-coded in the banner's inline `style` attribute. If that hard-coded padding ever changed, then this rule would need to also change.

Were you thinking of fixing the height in CSS (so also in `codered-front.scss`) rather than using the inline padding?